content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -210,6 +210,7 @@ The following tables list all supported operating systems with the name of platf
 | Ubuntu 18.04 (bionic) | `debian-openssl-1.1.x`   |  1.1.x  |
 | Ubuntu 19.04 (disco)  | `debian-openssl-1.1.x`   |  1.1.x  |
 | Ubuntu 20.04 (focal)  | `debian-openssl-1.1.x`   |  1.1.x  |
+| Ubuntu 22.04 (jammy)  | `debian-openssl-3.0.x`   |  3.0.x  |
```

Comment:
```suggestion
| Ubuntu 21.04 (hirsute)  | `debian-openssl-1.1.x`   |  1.1.x  |
| Ubuntu 22.04 (jammy)  | `debian-openssl-3.0.x`   |  3.0.x  |
```
I think this would be the value for 21.04?